### PR TITLE
Key removal

### PR DESCRIPTION
**Pull request by asteiker**
Moving s3credential guidance into markdown for security reasons. This will remove expired aws keys from the notebook.

----
📚 Documentation preview 📚: https://earthaccess--745.org.readthedocs.build/en/745/

### CODE REVIEW

File: docs/howto/edl.ipynb
[the remark this answers is not in the document]
Comment (asteiker) [reply docs/howto/edl.ipynb:1]: @MattF-NSIDC Totally fair. I'm good with the first approach to simply comment the code cell itself. Fwiw the whole notebook does run successfully even without this line.

[the remark this answers is not in the document]
Comment (mfisher87) [reply docs/howto/edl.ipynb:1]: Ah OK, then maybe we don't need to bold that note :)

[the remark this answers is not in the document]
Comment (asteiker) [reply docs/howto/edl.ipynb:1]: @MattF-NSIDC I moved it to a commented code line. Let me know if you think this is now ready to merge.

**Review by chuckwondo — APPROVED** [review]
This is good for now to get the credentials removed. Hopefully we won't have to worry about removing the entire history, which would be painful.

**Review by mfisher87 — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**